content/en/docs/reference/access-authn-authz/mutating-admission-policy.md

```diff
@@ -0,0 +1,364 @@
+title: Mutating Admission Policy
+content_type: concept
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="MutatingAdmissionPolicy" >}}
+
+This page provides an overview of _MutatingAdmissionPolicies_.
+
+<!-- body -->
+
+## What are MutatingAdmissionPolicies?
+
+Mutating admission policies offer a declarative, in-process alternative to mutating admission webhooks.
+
+Mutating admission policies use the Common Expression Language (CEL) to declare mutations to resources.
+Mutations can be defined either with an *apply configuration* that is merged using the
+[server side apply merge strategy](/docs/reference/using-api/server-side-apply/#merge-strategy)
+or a [JSON patch](https://jsonpatch.com/).
+
+Mutating admission policies are highly configurable, enabling policy authors to define policies
+that can be parameterized and scoped to resources as needed by cluster administrators.
+
+## What resources make a policy
+
+A policy is generally made up of three resources:
+
+- The MutatingAdmissionPolicy describes the abstract logic of a policy
+  (think: "this policy sets a particular label to a particular value").
+
+- A MutatingAdmissionPolicyBinding links the above resources together and provides scoping.
+  If you only want to set an `owner` label for `Pods`, the binding is where you would
+  specify this mutation.
+
+- A parameter resource provides information to a MutatingAdmissionPolicy to make it a concrete
+  statement (think "set the `owner` label to something like `company.example.com`").
+  A native type such as ConfigMap or a CRD defines the schema of a parameter resource.
+  MutatingAdmissionPolicy objects specify what kind they are expecting for their parameter resource.
+
+At least a MutatingAdmissionPolicy and a corresponding MutatingAdmissionPolicyBinding
+must be defined for a policy to have an effect.
+
+If a MutatingAdmissionPolicy does not need to be configured via parameters, simply leave
+`spec.paramKind` in  MutatingAdmissionPolicy not specified.
+
+## Getting Started with Mutating Admission Policy
+
+Mutating admission policy is part of the cluster control-plane. You should write
+and deploy them with great caution. The following describes how to quickly
+experiment with Mutating admission policy.
+
+### Create a MutatingAdmissionPolicy
+
+The following is an example of a MutatingAdmissionPolicy. This policy mutates newly created Pods to have a sidecar container if it does not exist.
+
+```yaml
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: MutatingAdmissionPolicy
+metadata:
+  name: "sidecar-policy.example.com"
+spec:
+  paramKind:
+    kind: Sidecar
+    apiVersion: mutations.example.com/v1
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["apps"]
+      apiVersions: ["v1"]
+      operations:  ["CREATE"]
+      resources:   ["pods"]
+  matchConditions:
+    - name: does-not-already-have-sidecar
+      expression: "!object.spec.initContainers.exists(ic, ic.name == \"mesh-proxy\")"
```

Comment:
Easier / more readable to use a YAML multiline string here, IMO.

Reply:
I agree for longer expressions but this seems reasonably contained, now that it's moved into a separate file